@here/harp-lines/test/rendering/RenderLines.ts

```diff
@@ -281,7 +281,7 @@ describe("Rendering lines: ", function() {
         }
 
         scene.position.set(-canvas.width / 2, -canvas.height / 2, 0);
-        webglRenderer = new THREE.WebGLRenderer({ canvas });
+        webglRenderer = new THREE.WebGL1Renderer({ canvas });
```

Comment:
with this change we require all our users to update to 0.118. I'm okay with this but if we want to support users on older versions of three (117?) we could replace this test with something like

webglRenderer = new ((THREE as any).WebGL1Renderer ?? THREE.WebGLRenderer)(…)